repos/spack_repo/builtin/packages/warpx/package.py

```diff
@@ -145,10 +145,16 @@ class Warpx(CMakePackage, PythonExtension):
             depends_on("pkgconfig", type="build")
     with when("compute=sycl"):
         depends_on("amrex +sycl")
+        depends_on("intel-oneapi-mkl", when="+fft")
```

Comment:
Thank you, @mabraham!

Yes, please feel free to add the same `+sycl` variant for our (`warpx` package) usage of `oneapi::mkl` in your PR #2550
```suggestion
        depends_on("intel-oneapi-mkl +sycl", when="+fft")
```